<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitVideoPanel</class>
 <widget class="QWidget" name="SplitVideoPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Split Video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Specify the time interval:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFrom">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teFrom">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbFrom">
       <property name="toolTip">
        <string>Set a current playback time</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTo">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teTo">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTo">
       <property name="toolTip">
        <string>Set a current playback time</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>146</red>
           <green>146</green>
           <blue>146</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSplit">
       <property name="text">
        <string>Trim</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSplit</sender>
   <signal>clicked()</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>startMencoder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>stopMencoder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFrom</sender>
   <signal>clicked()</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>setFromTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTo</sender>
   <signal>clicked()</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>setToTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teFrom</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>fromTimeChanged(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teTo</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>SplitVideoPanel</receiver>
   <slot>toTimeChanged(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopMencoder()</slot>
  <slot>setFromTime()</slot>
  <slot>setToTime()</slot>
  <slot>startMencoder()</slot>
  <slot>fromTimeChanged(QTime)</slot>
  <slot>toTimeChanged(QTime)</slot>
 </slots>
</ui>
